### DISCUSSION TITLE
PD powererd hub?

### DISCUSSION THREAD

**Discussion by sejtam** [outdoorbits/little-backup-box#216] [Q&A]
I am trying to use LBB with an external WD harddrive, but the Pi$B doesn ot have sufficient power itself

So I am trying to use a Belkin USB-C Hub  P-AVC009.html. which sadly (while supporting PD) does not seem to power the USB-A ports

My older Anker A8383 does power the disk, bt when I use it on the LBB, i cannot see the USB card reader nor the disk inserted into it.

is there a known USB-c/PD hub that that powers  the USB-A ports, *and* can be used with LBB

**Comment by outdoorbits** [comment]
As described at 
https://github.com/outdoorbits/little-backup-box/wiki/02.-Hardware - 
Useful accessories ORICO 4-Port USB3.0 HUB works fine, I`m using it. But 
following the link I found the orico shop inactive. The UGREEN 4-Port 
USB 3.0 HUB also seems not to be available anymore. I just successfully 
tested Anker 60W 7-Port USB 3.0 Data Hub 
https://www.amazon.de/Anker-Datenhub-PowerIQ-Ladeports-Motorola/dp/B00ZQFYWJY?language=en_GB&currency=EUR 
.

**Comment by outdoorbits** [comment]
Please give feedback if you found a working device. I would like to update the wiki.

**Comment by sejtam** [comment]
But that doesn't seem to  external pdin for separate power supply from the Pi?


***@***.***


On 8 July 2023, 13:58, outdoorbits wrote:


As described at 
https://github.com/outdoorbits/little-backup-box/wiki/02.-Hardware - 
Useful accessories ORICO 4-Port USB3.0 HUB works fine, I`m using it. But 
following the link I found the orico shop inactive. The UGREEN 4-Port 
USB 3.0 HUB also seems not to be available anymore. I just successfully 
tested Anker 60W 7-Port USB 3.0 Data Hub 
https://www.amazon.de/Anker-Datenhub-PowerIQ-Ladeports-Motorola/dp/B00ZQFYWJY?language=en_GB&currency=EUR 
.

**Comment by outdoorbits** [comment]
Of course, look at the images, there is a 60 W power supply.

**Comment by sejtam** [comment]
Hmm

I'm not in DE and it says that product id disn't exist, so i searched 
and found a similar one which didn't have the adapter

Byt anyway, i found an online shop locally that says they have the ORICO
one in stock, (fpr SGD 15, good deal), so I ordered one.

**Comment by thelightdecides** [comment]
I just tested the ugreen hub with a HDD and a CFexpress B card reader connected. Powered via USB powerbank. No errors.
https://www.amazon.de/dp/B07M5LMGXZ?psc=1&ref=ppx_yo2ov_dt_b_product_details
